lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -84,6 +84,21 @@ def stub_publishing_api_publish(content_id, body, response_hash = {})
         stub_request(:post, url).with(body: body).to_return(response)
       end
 
+      # Stub a POST /v2/content/:content_id/republish request
+      #
+      # @param content_id [UUID]
+      # @param body  [String]
+      # @param response_hash [Hash]
+      def stub_publishing_api_republish(content_id, body = {}, response_hash = {})
+        url = PUBLISHING_API_V2_ENDPOINT + "/content/#{content_id}/republish"
+        response = {
+          status: 200,
+          body: '{}',
+          headers: { "Content-Type" => "application/json; charset=utf-8" }
+        }.merge(response_hash)
+        stub_request(:post, url).with(body: body).to_return(response)
+      end
```

Comment:
This looks good too. I think we'd also want one of stub_any_publishing_api_republish for consistency.

There's also the `assert*` methods too but I guess we can just add them if someone wants them as they seem like something that's not too likely to be needed.

Reply:
Yep good point